modules/bismark/methylationextractor/main.nf

```diff
@@ -31,6 +31,13 @@ process BISMARK_METHYLATIONEXTRACTOR {
     path "versions.yml"                            , emit: versions
 
     script:
+    // Assign sensible numbers for multicore and buffer_size based on bismark docs
+    def ccore = task.cpus ? ((task.cpus as int) / 3) as int : 1
+    def multicore = (ccore > 1) ? "--multicore ${ccore}" : ""
```

Comment:
```suggestion
    def multicore = (task.cpus >= 6) ? "--multicore ${(task.cpus / 3) as int}" : ""
```

Reply:
Same as memory, this will break if cpus is `null` right? Not that I've ever actually seen that happen in the wild.

Reply:
Even if it happens, `(null >= 6)` evaluates to `false`